web/components/DayBlock.jsx

```diff
@@ -10,12 +10,12 @@ import theme from '../styles/theme'
 
 class DayBlock extends React.Component {
 
-    renderTinySetBlocks = (setBlocks, day) => {
+    renderTinySetBlocks = (setBlocks, day, fetchingData) => {
```

Comment:
Shouldn't this be refactored to use `pendingNetworkCalls` instead of `fetchingData`?

Reply:
Yes, my mistake I overlooked it, I want to map fechingData with `environment.pendingNetworkCalls > 0 `